### PR TITLE
Feature/api post

### PR DESCRIPTION
**Pull request by darlaevans2000**
# Thank you for pushing your code!
## What (if any) features are you implementing?
- Add api post
## What (if anything) did you refactor?
- n/a
## Were there any issues that arose?
- some integers messed up all users avg
## Is there anything that you need from your teammate?
- parse float
## Any other comments, questions, or concerns?
- n/a
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?
- [X] open index.html
- [ ] console.log()
- [ ] dev tools

## Checklist:
- [X] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -7,192 +7,226 @@
     <title>FitLit Activity Tracker</title>
     <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans:400,800&display=swap"
     rel="stylesheet" />
-  </head>
```
Comment (williamphelps13) [inline dist/index.html:10]: Good Fix on the Merge Conflict

File: dist/index.html [hunk 2]
```diff
@@ -7,192 +7,226 @@
...
+          <div class="card-history-hydration">
+            <p class="historicalWeek"></p>
+            <div class="sidebar-header-line"></div>
+            <ul class="card-vertical-list" id="hydrationEarlierWeek">
+              <!-- Hydration week list goes here -->
+            </ul>
+          </div>
         </section>
       </section>
-      <section class="body-main-infoContainter">
```
Comment (williamphelps13) [inline dist/index.html:45]: Adding the info containers to the top definitely helped minimize the impact on the grid layout and responsiveness

**Review by MTCawthray — APPROVED** [review]
Looks fabulous! We can move the query selectors from the scripts file over to the domUpdates file easily. Also, we will add a parseFloat() to some of the data in the post bodies that is screwing up our numbers :)

File: src/apiCalls.js [hunk 3]
```diff
@@ -1,3 +1,53 @@
 export function fetchData(file) {
   return fetch(`http://localhost:3001/api/v1/${file}`).then(response => response.json());
-}
\ No newline at end of file
+}
+
+export function postHydrationData(userId, date, amount){
```
Comment (williamphelps13) [inline src/apiCalls.js:5]: Awesome work with the post functions! Really great!

**Review by williamphelps13 — APPROVED** [review]
This was hard, and you did a great job! I'll work on those changes that need to be added that you noted!

**Pull request merged**